### PR TITLE
{vis}[GCCcore/10.3.0] Ghostscript v9.54.0, ImageMagick v7.0.11-14, LittleCMS v2.12

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/0efdc2e3eec556343fe692a71e1985a7 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/fca2578527043a5f2090335c5c1aebe9 for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 2 out of 4 (3 easyconfigs in total)
bask-pg0308u09a - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Platinum 8358P CPU @ 2.60GHz (icelake), Python 3.6.8
See https://gist.github.com/3eaf52d30f67448e710072732a7d2a49 for a full test report.

**Comment by branfosj** [comment]
> 
> 
> Test report by @branfosj
> **FAILED**
> Build succeeded for 2 out of 4 (3 easyconfigs in total)
> bask-pg0308u09a - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Platinum 8358P CPU @ 2.60GHz (icelake), Python 3.6.8
> See https://gist.github.com/3eaf52d30f67448e710072732a7d2a49 for a full test report.

I've seen an early failure in building Ghostscript, which looks to be a race condition. Setting `parallel = 1` fixes it. Done in 54815a3

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/53eb8716f76bfc7072a8f207321c322f for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (3 easyconfigs in total)
bask-pg0308u09a - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Platinum 8358P CPU @ 2.60GHz (icelake), Python 3.6.8
See https://gist.github.com/425ef68cadef050504c3c9d171e607f0 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 15 out of 26 (3 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/0a1d9df13c8c3bae6fa67a904da330bd for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 9 out of 9 (3 easyconfigs in total)
node372.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/d7fb9b03cabb982083281d8b33a8af8e for a full test report.

File: easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.54.0-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    ('pkg-config', '0.29.2'),
+]
+
+# Do not use local copies of zlib, jpeg, freetype, and png
+preconfigopts = "mv zlib zlib.no && mv jpeg jpeg.no && mv freetype freetype.no && mv libpng libpng.no && "
+preconfigopts += 'export LIBS="$LIBS -L$EBROOTZLIB/lib -lz" && '
+
+configopts = "--with-system-libtiff --enable-dynamic"
+
+parallel = 1
```
Comment (lexming) [inline easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.54.0-GCCcore-10.3.0.eb:42]: @branfosj I had no issues running this build with 4 cores. See https://gist.github.com/f0ad62fd86c26a711d51c19c0a94fcbd

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.54.0-GCCcore-10.3.0.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.54.0-GCCcore-10.3.0.eb]: I saw the issue when using 8 cores, so I've switched to `maxparallel = 4` in 176c485.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c1667236597792dbc80a36ef8021fa23 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @branfosj !

**Pull request merged**